docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -167,6 +167,14 @@ When enabling metrics for {project_name}'s embedded caches, the metrics now use
 For more details, check the
 link:{upgradingguide_link}[{upgradingguide_name}].
 
+= Brute Force Protection changes
+
+There have been a couple of enhancements to the Brute Protection:
+
+1. When an attempt to authenticate with an OTP or Recovery Code fails due to Brute Force Protection the active Authentication Session is invalidated. Any further attempts to authenticate with that session will fail.
+
+2. In previous versions of Keycloak the Administrator had to choose between whether to disable users temporarily or permanently due to a Brute Force attack on their account. The administrator now has the option to permanently disable a user after a given number of temporary lockouts.
```

Comment:
We also need to update https://www.keycloak.org/docs/latest/server_admin/#password-guess-brute-force-attacks, right?